core/tests/gae_suite.py

```diff
@@ -39,7 +39,7 @@
         OPPIA_TOOLS_DIR, 'google_appengine_1.9.67', 'google_appengine',
         'lib', 'webob_0_9'),
     os.path.join(OPPIA_TOOLS_DIR, 'browsermob-proxy-0.7.1'),
-    os.path.join(OPPIA_TOOLS_DIR, 'selenium-2.53.2'),
+    os.path.join(OPPIA_TOOLS_DIR, 'selenium-3.14.1'),
```

Comment:
I went through the changelog for selenium 3.14.1. This seems to be a major upgrade. Can you please test whether the e2e tests (any one of them) works fine locally? Also, please remove oppia tools dir before this.

Reply:
Yep, they do work fine; I checked.

Reply:
Thanks for checking this!